tbx/project_styleguide/templates/patterns/molecules/author/author.html

```diff
@@ -2,7 +2,7 @@
 
 <div class="author{% if classes %} {{ classes }}{% endif %}">
 
-    {% include "patterns/atoms/avatar/avatar.html" with avatar=author.image classes="avatar--small author__avatar" %}
+    {% include "patterns/atoms/avatar/avatar.html" with avatar=author.image classes="avatar--"|add:avatar_size|add:" author__avatar" %}
```

Comment:
Q: Is the second `|add:` needed?

Reply:
I believe so, because otherwise we can't finish the string around the classnames correctly, and the last `author__avatar` part of the string is considered a variable.

Reply:
Ahh, I'm with you. I guess you could do `classes="author__avatar avatar--"|add:avatar_size` but depends on the order in which you want the classes added  : )

Reply:
Oh you're right - but I'm going to leave this now due to time pressure.